HIP-1.md

```diff
@@ -0,0 +1,146 @@
+| hip | title               | status | author | created    |
+| --- | ------------------- | ------ | ------ | ---------- |
+| 1   | Sovereign consensus | Draft  | asaj   | 2022-10-04 |
+
+### **Brief Summary / Abstract**
+
+Defines the specification for sovereign consensus, which puts control of the interchain security model in the hands of message recipients.
+
+### **Motivation**
+
+Interchain security models offer different tradeoffs with respect to security, latency, and cost. For example:
+
+1. Multisig models prioritize latency and cost over security
+2. Bonding models (e.g. proof-of-stake) prioritize latency and security over cost
+3. Optimistic models prioritize security and cost over latency
+4. Natively verified models prioritize security over cost (and sometimes latency)
+
+Interchain applications necessarily adopt the tradeoffs of their interchain security models. Rather than offering a one-size-fits-all product, sovereign consensus allows applications to choose (or construct) a security model that offers the best choice of tradeoffs for that application.
+
+Furthermore, the modularity of sovereign consensus allows Hyperlane to be forward compatible with new security models as they emerge.
+
+### **Tech Spec**
+
+Sovereign consensus is powered by `Interchain Security Modules` or `ISMs`, which specify the rules by which a `recipient` will accept an interchain message.
+
+Message `recipients` should specify their `ISM` via the `interchainSecurityModule()` view function.
+
+The `Inbox` contract, when delivering a message via `Inbox.process()`, calls `recipient.interchainSecurityModule().accept()`, which accepts or rejects the message.
+Relayers can inject arbitrary off-chain data (e.g. validator signatures, zero-knowledge proofs, etc.) into `accept()`, allowing for a wide degree of flexibility in `ISM` design.
```

Comment:
I assume the separation of `interchainSecurityModule()` and `accept()` is such that ISMs can be reused across applications. Worth noting there is one extra view call than strictly necessary.

Reply:
instead of having `accept()` directly on the recipient